src/implementation/Server/HTTPServer/HTTPServer.ts

```diff
@@ -11,40 +11,54 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import Restana from "restana";
+import * as http from "http";
+import express from "express";
 import bodyParser from "body-parser";
 import HTTPServerImpl from "./HTTPServerImpl";
 import IServer from "../../../interfaces/Server/IServer";
 import { DaprClient } from "../../..";
 import { createHttpTerminator } from "http-terminator";
 import { Logger } from "../../../logger/Logger";
+import { DaprServerOptions } from "../../../types/DaprServerOptions";
 
-// eslint-disable-next-line
-export interface IServerImplType extends HTTPServerImpl {}
-// eslint-disable-next-line
-export interface IServerType extends Restana.Service<Restana.Protocol.HTTP> {}
+export type IServerImplType = HTTPServerImpl;
+export type IServerType = express.Express;
 
 export default class HTTPServer implements IServer {
+  server: IServerType;
+  serverInstance: undefined | http.Server; // defined after start()
   serverHost: string;
   serverPort: string;
-  isInitialized: boolean;
-  server: IServerType;
   serverAddress: string;
+  serverOptions: DaprServerOptions;
   serverImpl: IServerImplType;
+  isInitialized: boolean;
   client: DaprClient;
   private readonly logger: Logger;
 
-  constructor(client: DaprClient) {
+  constructor(client: DaprClient, options: DaprServerOptions) {
     this.serverHost = "";
     this.serverPort = "";
+    this.serverOptions = options;
     this.client = client;
     this.logger = new Logger("HTTPServer", "HTTPServer", client.options.logger);
 
     this.isInitialized = false;
 
-    this.server = Restana();
-    this.server.use(bodyParser.text());
-    this.server.use(bodyParser.raw());
+    this.server = express();
```

Comment:
Can we add a DaprServerOption to allow users to pass their express objects? We can clearly mention that takes precedence over options like maxBodySizeMb

Reply:
I do like that idea! Will add!!